Validation/HcalDigis/src/HcalDigisClient.cc

```diff
@@ -174,35 +179,47 @@ int HcalDigisClient::HcalDigisEndjob(const std::vector<MonitorElement*> &hcalMEs
         }
     
         //zero the sumphi vector at the start of each ieta ring
-	for(int depth = 1; depth <= depths; depth++) sumphi[depth-1] = 0;
+    for(int depth = 1; depth <= depths; depth++) {
```

Comment:
can be sumphi.assign(depths,0)

Reply:
also the indentation of this loop seems to be incorrect (relative to the lines above). Please make it consistent

Reply:
Thanks @davidlange6 - they should be looking better now.